data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -22,34 +22,35 @@ class TokenSettings(ZkLendSpecificTokenSettings, TokenSettings):
     pass
 
 
-def convert_string_to_decimal_1e27(string_big_num):
+def format_reserve_data_number(string_large_num: str):
     """
-    Method to convert big string number to decimal.
+    Method to convert large string number to decimal.
     Example:
-    string_big_num = "800000000000000000000000000"
+    string_large_num = "800000000000000000000000000"
     return = 0.80
     """
-    decimal_big_num = decimal.Decimal(string_big_num)
-    scale_factor = decimal.Decimal("1e27")
-    bigResult = decimal_big_num / scale_factor
-    # Have to cast "0.800000000000000000000000000" to "800000000000000000000000000"
-    formatted_number = "{:.2f}".format(bigResult)
+    int_large_num = decimal.Decimal(string_large_num)
```

Comment:
1) This is not an integer.
2) Avoid naming variables based on their type as much as possible. That's what type hints are for 😉

Reply:
fix it in this commit 73f712a53242c2402dbad84e335c8b84ae1e3ebf
i change the variables names to use something more general